test/AutoRest.TestServer.Tests/Common/StringExtensionsTests.cs

```diff
@@ -21,6 +21,7 @@ public class StringExtensionsTests
         [TestCase("sheep", "sheep")]
         [TestCase("people", "people", false)]
         [TestCase("child", "children")]
+        [TestCase("data", "data", false)]
```

Comment:
Can you also add cases for `"datatip"`, `"tipdata"`, `"da_ta"`?

Reply:
Added. As I only customized for `data`, all these words would use the original rules from the `Humanize` library which does not handle combinations of nouns well. While `RenameGetMethod` and `RenameListToGet` will use the private methods which extract the last word and transform it as expected. I also added tests fro them.